appcheck/app/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id 'com.android.application'
+}
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdkVersion 30
+
+    defaultConfig {
+        applicationId "com.google.firebase.example.appcheck"
+        minSdkVersion 16
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+}
+
+dependencies {
+
+    implementation 'androidx.appcompat:appcompat:1.3.1'
+    implementation 'com.google.android.material:material:1.4.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+
+    implementation 'com.google.firebase:firebase-appcheck:16.0.0-beta02'
+    implementation 'com.google.firebase:firebase-appcheck-debug:16.0.0-beta02'
+    implementation 'com.google.firebase:firebase-appcheck-safetynet:16.0.0-beta02'
+
+    implementation 'com.squareup.retrofit2:retrofit:2.9.0'
+
+    implementation "androidx.core:core-ktx:1.6.0"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+}
+repositories {
+    mavenCentral()
+}
```

Comment:
This may have changed by I think we should have `apply plugin: 'com.google.gms.google-services'` here. Unless we are creating the app directly in the app.